### DISCUSSION TITLE
What is impact of increasing misfireThresold value?

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2247] [Q&A]
I’m using quartz.net 3.5. We see lot of misfires. Right now the value of misfireThresold value is 60 seconds. In production minimum recurring time for a job is 30 minutes.

so I’m thinking to increase the value of 30 minutes or 20 minutes. 

But before I make this change, I would like to know is there any problem with increasing this value with such a high value . 

By default if a job is missed, it’s retried within the misfireThresold value based on the configured value of misfire instruction. Also I would like to know that how does this work? Does it keep polling the database within this time period? 

Just FYI sometimes there could be 2000 jobs scheduled per second.

**Comment by viveknuna** [comment]
@lahma do you have any inputs on this?

**Reply by viveknuna** [reply]
@jafin do you have inputs on this!

**Reply by jafin** [reply]
@viveknuna I don't have direct inputs, I would suggest attempting to isolate the issue into a runnable test, and spike your idea above and evaluate the outcome. From there you could make an informed decision as to what the outcome may be.
